src/tools/sdk/_windows.md

```diff
@@ -1,11 +1,13 @@
+How to install?
+
```

Comment:
I think we'd be fine without this, as the section of the page this is in is called "Installing the Dart SDK". 

```suggestion
```